SB2018/2018-02-27.md

```diff
@@ -0,0 +1,26 @@
+---
+to: All Teams
+subject: Competiton dates confirmed! (And some other bits)
+---
+
+Hello all!
+
+I'm happy to confirm the dates for the competition this year! The competition will be held from *Saturday 21st April* until *Sunday 22nd April*. The competition will be held at the University of Southampton's Highfield campus, so please plan accommodation / transport accordingly. The exact start/end times are to be confirmed, but it'll start at some point in the morning of the 21st, and finish in the afternoon of the 22nd, no later than 5:00pm.
+
+This information will be reflected on our website soon!
+
+## Competition Progress
+
+You're now just over half way through this competition period. Hopefully you're making good progress with your robot, and are well on your way to having something competition ready!
+
+## Tech Days
+
+There are just 2 remaining tech days between now and the competition. These are a great chance to spend a full day testing on your kit, and get direct support from us. The remaining tech days are:
+
+- 17th March
+- 14th April
+
+Both of these are likely to be held at Highfield campus, as with the last 2, however confirmation of this will come closer to the time.
+
+## Media Consent
+As with Student Robotics competition, media consent will be required to attend the competition. The forms for this will be sent through in the next week or two. These will need to be signed by each member of your team, as well as staff. Those under 18 will need a parent / guardian to sign on their behalf.
```

Comment:
Remove the 'As with the Student Robotics Competition', one thing we mentioned at kickstart was we shouldn't reference SR as much as we did.